packages/core/src/components/Dropdown/Dropdown.test.jsx

```diff
@@ -0,0 +1,181 @@
+import { mount, shallow } from 'enzyme';
+import Dropdown from './Dropdown';
+import React from 'react';
+
+const defaultProps = { name: 'dropdown', label: 'Select an option' };
+
+/**
+ * Generate <option> elements
+ * @param {number} count
+ */
+function generateOptions(count) {
+  const options = [];
+
+  for (let i = 1; i < count + 1; i++) {
+    options.push({
+      value: String(i),
+      label: String(i)
+    });
+  }
+
+  return options;
+}
+
+/**
+ * Helper method for shallow rendering the <Dropdown> component. The only props
+ * initially defined are the required props.
+ * @param {object} customProps - Additional props
+ * @param {number} optionsCount - Total number of <option>'s
+ * @return {object}
+ */
+function render(customProps = {}, optionsCount = 1, deep = false) {
+  const props = { ...defaultProps, ...customProps };
+  const component = (
+    <Dropdown {...props} options={generateOptions(optionsCount)} />
+  );
+
+  return {
+    props: props,
+    wrapper: deep ? mount(component) : shallow(component)
+  };
+}
+
+describe('Dropdown', () => {
+  it('renders a select menu', () => {
+    const data = render();
+
+    expect(data.wrapper.find('FormLabel').length).toBe(1);
+    expect(data.wrapper.find('select').length).toBe(1);
+    expect(data.wrapper.find('select').children('option').length).toBe(1);
+    expect(data.wrapper).toMatchSnapshot();
```

Comment:
Might be good to have one of these assign a `value` or `defaultValue` so we can see how that looks on the final output.